build/node.sh

```diff
@@ -2,9 +2,18 @@
 
 set -e
 
-if [ -z ${NODE_VERSION+x} ]; then echo "No NODE_VERSION defined - skipping" && exit; else echo "Installing node $NODE_VERSION"; fi
+if [ -z ${NODE_VERSION+x} ]; then echo "No NODE_VERSION defined - skipping" && exit; fi
 
-curl -sL https://deb.nodesource.com/setup_${NODE_VERSION}.x | bash
+SEMVER_REGEX="^(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))?(\.(0|[1-9][0-9]*))?$"
+
+if ! [[ "$NODE_VERSION" =~ $SEMVER_REGEX ]]; then
+  echo Not a semver tag - skipping: ${NODE_VERSION}
+  exit
+fi
+
+echo "Installing node $NODE_VERSION";
+
+curl -sL https://deb.nodesource.com/setup_${BASH_REMATCH[1]}.x | bash
```

Comment:
This allows us to pass the full node version, so we can switch install method later more easy